### ISSUE TITLE
[SCIM] Proposal: Load SCIMSchema from string

### ISSUE THREAD

**Issue by johanndev** [simpleidserver/SimpleIdServer#763]
Currently, the `SCIMSchemaExtractor.Extract()` method expects a file path as input, which is immediately read and parsed into a JObject:

https://github.com/simpleidserver/SimpleIdServer/blob/2c65b5256e1a902db0948395c59819342e23178d/src/Scim/SimpleIdServer.Scim/SCIMSchemaExtractor.cs#L14-L19

It would be convenient if there was a separate method that would accept a string directly instead of a filepath. That would make it easier to load schemas from other sources, e.g., Resource Files.

If you think this a worthwile addition, I could provide a proposal in a PR.

Thanks!

**Comment by simpleidserver** [comment]
Indeed, it would be useful to have another method that accepts a string as input. I'd be glad if you could make a pull request. :)
